src/library/dao/macrodao.cpp

```diff
@@ -1,4 +1,4 @@
-#include "macros/macrodao.h"
+#include "macrodao.h"
```

Comment:
Please do not use relative include paths!

```suggestion
#include "library/dao/macrodao.h"
```

Reply:
CLion does that automatically, and unfortunately I haven't yet found an option for the headers. I configured it correctly for all other includes though, but it sees the corresponding header as special.